src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -25,3 +25,33 @@ admin_products_v2_edit:
       id_product: productId
   requirements:
     productId: \d+
+
+admin_products_v2_get_images:
+    path: /{productId}/getImages
+    methods: [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:getImages
+        _legacy_controller: AdminProducts
+        _legacy_link:
+            - AdminProducts:getimages
+        _legacy_parameters:
+            id_product: productId
+    options:
+        expose: true
+    requirements:
+        productId: \d+
+
+admin_products_v2_add_image:
+    path: /{productId}/addImage
```

Comment:
```suggestion
    path: /{productId}/add-image
```

Reply:
How about
GET /{productId}/images
POST /{productId}/images/add?

Reply:
I like that last one better